### DISCUSSION TITLE
Openchangelog now supports git-cliff changelogs

### DISCUSSION THREAD

**Discussion by JonasHiltl** [orhun/git-cliff#947] [Show and tell]
Hi, I'm a big fan of `git-cliff`!

I created [Openchangelog](https://github.com/JonasHiltl/openchangelog), an open source tool that turns your `CHANGELOG.md` into a website (as long as it follows the keep a changelog format).

Just pushed a [minor fix](https://github.com/JonasHiltl/openchangelog/commit/7f03379edf2176e6a5da670aabba02ce87e9dc3a) today that adds full support for changelogs generated with git-cliff. You can check out how it looks with the git-cliff changelog here: https://git-cliff.openchangelog.com/

I've also added git-cliff to our [docs](https://openchangelog.com/docs/getting-started/conventional-commits/) as a recommended tool for generating `CHANGELOG.md` files from conventional commits.

It would be awesome if you could add the openchangelog link to the `README` - would help others discover this integration!
Maybe in the `Similar/Related Projects` section.

**Comment by orhun** [comment]
Wow, that's awesome! I'd be happy to mention Openchangelog in README (also feel free to submit a PR!)

Question: is this a platform to _maintain_ changelog for projects? For the longest time I was planning to create a website for generating changelogs using `git-cliff` but didn't have the necessary web skills to do it. Maybe there is a collaboration opportunity there :)

**Reply by JonasHiltl** [reply]
It's not used to maintain it, it just takes your `CHANGELOG.md` as input and renders a website for it. I built a GitHub integrations so that the site stays in sync with the changelog on GitHub. You still have to maintain it _yourself_ or automate it using for example `git-cliff`.

It's also not a static site generator, since the changelog is fetched and parsed on every request to the web server, but you can configure caching to speed up the requests and not hit GitHub rate limits.

**Reply by orhun** [reply]
Ah I see, pretty cool 👍🏼
